src/afl-fuzz-stats.c

```diff
@@ -385,7 +385,7 @@ void maybe_update_plot_file(afl_state_t *afl, u32 t_bytes, double bitmap_cvg,
                  afl->plot_prev_md == afl->max_depth &&
                  afl->plot_prev_ed == afl->fsrv.total_execs) ||
                 !afl->queue_cycle ||
-                get_cur_time() - afl->start_time <= 60000))) {
+                get_cur_time() == afl->start_time))) {
```

Comment:
I'm not familiar with the details, but is equal too "strict"? What if this functions isn't called immediately at `start_time` but at `start_time + 1`?  Please verify this before merging.

Reply:
The condition is for when **not to** append data to plot data.

Here is the output.

```
# relative_time, cycles_done, cur_item, corpus_count, pending_total, pending_favs, bit_cvg, shw_cvg, saved_crashes, saved_hangs, max_depth, execs_per_sec, total_execs, edges_found
0, 0, 2, 32, 32, 29, 59.680%, 0.225%, 0, 0, 1, 0.00, 641, 39112
5, 0, 2, 112, 112, 29, 66.035%, 0.254%, 0, 0, 2, 376.62, 3024, 43277
10, 0, 2, 184, 184, 29, 67.233%, 0.256%, 0, 0, 2, 335.62, 4566, 44062
15, 0, 2, 247, 247, 29, 67.581%, 0.258%, 0, 0, 2, 301.83, 5905, 44290
20, 0, 2, 311, 311, 29, 67.831%, 0.261%, 0, 0, 2, 300.17, 7274, 44454
25, 0, 2, 378, 378, 29, 68.022%, 0.262%, 0, 0, 2, 308.25, 8708, 44579
30, 0, 2, 447, 447, 29, 68.340%, 0.263%, 0, 0, 2, 295.22, 10197, 44787
...
```